common/changes/@itwin/tree-widget-react/rework_additional_trees_handling_2023-04-14-14-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/tree-widget-react",
+      "comment": "**BREAKING:** Reworked how trees are provided to the widget. Instaed of having separate configuration properties for hiding default trees and adding additional trees now there is only one property for supplying a list of trees to show.",
```

Comment:
```suggestion
      "comment": "**BREAKING:** Reworked how trees are provided to the widget. Instead of having separate configuration properties for hiding default trees and adding additional trees now there is only one property for supplying a list of trees to show.",
```